episodes/02-setup.md

```diff
@@ -17,6 +17,12 @@ exercises: 0
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
+:::::::::::::::::::::::::::::::::::::::: instructor
+
+There are no slides for this episode. Explain what a command line is, why it is useful, and why we use it in this workshop. Participants are often new to the command line and don't get why we not use a git gui. Only focus on the bare essentials for setting up git. We shall use nano editor so that everyone is on the same page.
```

Comment:
Can you actually remove all the commands that we usually do not teach?

Reply:
Yes, I have created a separate issue for this. See https://github.com/esciencecenter-digital-skills/good-practices-in-research-software-development/issues/47